### PR TITLE
support live preview in AS 4.2

### PR DESCRIPTION
**Pull request by azizimusa**
added attributeset so that live preview works (only in android studio 4.2)

Screenshot 2020-09-29 at 10 47 35 AM

### CODE REVIEW

File: sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt [hunk 1]
```diff
@@ -33,10 +34,15 @@ import com.squareup.contour.ContourLayout
 import com.squareup.picasso.Picasso
 
 @SuppressLint("SetTextI18n")
-class ExpandableBioCard1(context: Context) : ContourLayout(context) {
+class ExpandableBioCard1(context: Context, attributeSet: AttributeSet? = null) : ContourLayout(context) {
```
Comment (saket) [inline sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt:37]: Let's rename the params to `attrs` and pass them to `ContourLayout`'s constructor as well?

File: sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt [hunk 2]
```diff
@@ -33,10 +34,15 @@ import com.squareup.contour.ContourLayout
...
   private val avatar = CircularImageView(context).apply {
     scaleType = CENTER_CROP
-    Picasso.get().load("https://upload.wikimedia.org/wikipedia/en/9/92/BenSisko.jpg").into(this)
+
+    //image will appear on emulator and devices
```
Comment (saket) [inline sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt:41]: This comment isn't necessary, the code is self-documenting.

Comment (saket) [reply sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt:41]: Same for the comment in `ExpandableBioCard2`

File: sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt (hunk 1 above)
Comment (saket) [reply sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt:37]: Same for `ExpandableBioCard2` and `SampleView`.

File: sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt [hunk 3]
```diff
@@ -33,10 +34,15 @@ import com.squareup.contour.ContourLayout
...
+    if (!isInEditMode)
+    Picasso.get()
+            .load("https://upload.wikimedia.org/wikipedia/en/9/92/BenSisko.jpg")
+            .into(this)
```
Comment (saket) [inline sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt:45]: Can you revert this formatting change?

**Review by saket — CHANGES_REQUESTED** [review]
Let's also add `Color.GRAY` as the background to the ImageViews in edit mode so that they look better in preview?

File: sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt [hunk 4]
```diff
@@ -33,10 +34,15 @@ import com.squareup.contour.ContourLayout
...
 @SuppressLint("SetTextI18n")
-class ExpandableBioCard1(context: Context) : ContourLayout(context) {
+class ExpandableBioCard1(context: Context, attributeSet: AttributeSet? = null) : ContourLayout(context) {
   private val avatar = CircularImageView(context).apply {
     scaleType = CENTER_CROP
-    Picasso.get().load("https://upload.wikimedia.org/wikipedia/en/9/92/BenSisko.jpg").into(this)
+
+    //image will appear on emulator and devices
+    if (!isInEditMode)
+    Picasso.get()
```
Comment (JakeWharton) [inline sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt:43]: Needs indented. Also please use `{` and `}` on the `if`.

File: sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard2.kt [hunk 5]
```diff
@@ -33,9 +34,12 @@ import com.squareup.contour.ContourLayout
...
 
 @SuppressLint("SetTextI18n")
-class ExpandableBioCard2(context: Context) : ContourLayout(context) {
+class ExpandableBioCard2(context: Context, attributeSet: AttributeSet? = null) : ContourLayout(context) {
   private val imageView = ImageView(context).apply {
     scaleType = CENTER_CROP
+
+    //image will appear on emulator and devices
+    if (!isInEditMode)
     Picasso.get()
```
Comment (JakeWharton) [inline sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard2.kt:43]: Needs indented now. Also please use `{` and `}` on the `if`.

File: sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt (hunk 3 above)
Comment (saket) [reply sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt:45]: Resolving this in favor of https://github.com/cashapp/contour/pull/110#discussion_r496350121

File: sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt (hunk 2 above)
Comment (azizimusa) [reply sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt]: done

**Review by saket — APPROVED** [review]

**Pull request merged**